airgun/widgets.py

```diff
@@ -938,9 +943,9 @@ def read(self):
         """
         parameters = []
         for row in self.rows():
-            name = row['Name'].widget.read()
-            value = row['Value'].widget.read()
-            parameters.append({'name': name, 'value': value})
+            name = row[self.name].widget.read()
+            value = row[self.value].widget.read()
+            parameters.append({self.name_key: name, self.value_key: value})
```

Comment:
Are `name_key` and `value_key` doing anything functionally here, other than replacing the use of the string `'name'` and `'value'`?

Reply:
I think it just clicked that all of this is being done to work around the fact that there's a table that is including the sort character.

That doubles my commitment to the idea that we should push back on UX and exclude the sorting indicator from the text content of the column.

Reply:
Yes because of the hard coding done on the table name this was not reusable, I was just trying to make that reusable.